src/components/Landing/Faq.tsx

```diff
@@ -7,7 +7,7 @@ export default function Faq() {
   return (
     <Collapse defaultActiveKey={QAs.length ? 0 : undefined} accordion>
       {QAs().map((qa, i) => (
-        <CollapsePanel header={qa.q} key={i}>
+        <CollapsePanel header={qa.q} key={`qa_panel_${i}`}>
```

Comment:
I think this is a step in the right direction, but will it be strong enough? I don't have the full breakdown to how much data is in each list and how performant each should be but this is a good writeup about it:
https://www.developerway.com/posts/react-key-attribute

I'm a bit hungover so i'm only taking in about 40% of it, but will re-read tomorrow when I am healthy again.

TLDR is I think adding `qa_panel_` to the front is the same in terms of performance as just having `i` as it will treat it as the same prop (if that makes sense). The benefit of using a unique id for each record (UUID, or some other unique id assoc with record) is that irrespective of whether the indexing changes, it will not destroy and recreate elements, but instead just smartly move them around (I think)

Reply:
Totally totally, thanks for calling it out!

That was a great article - I didn't know that `key` values don't need to be globally unique; just unique for each specific tree (loop).

In this case, I think using the index is fine. `QAs()` is a static list; the order of items, or the items themselves, won't ever change, so I don't think we need to worry about unique ids for keys.

I've gone over every `key` prop in the app and checked if it's a dynamic (meaning, the elements might change) or static list; in all possible cases I've tried to use unique keys based on the data itself, rather than indexes. But for static lists, I've left those as indexes.

Let me know if I'm misunderstanding something though (or if I've missed anything in the code). Def want to get this right 🙏 .

Reply:
Nah I think that’s it! I’ll need to double check the article I linked but it sounds like the same understanding I had too

Reply:
Same with this text

Reply:
done! https://github.com/jbx-protocol/juice-interface/pull/1135#discussion_r889677226